Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -69,24 +69,6 @@ Install the Cilium CLI
 
 .. include:: ../../installation/cli-download.rst
 
-Starting from v0.14.8, the Cilium CLI supports configuring Cilium Cluster Mesh
-using Helm. To opt in to use the new *helm* configuration mode export the
-``CILIUM_CLI_MODE`` variable:
-
-.. code-block:: shell-session
-
-  export CILIUM_CLI_MODE=helm
-
-Always prefer the Cilium CLI *helm* mode for new Cluster Mesh installations, as
-it supports more advanced functionalities and configuration options. It also enables
-you to use the Cilium CLI and Helm interchangeably to manage your Cilium installation.
-
-.. warning::
-
-  Don't use the Cilium CLI *helm* mode to enable Cluster Mesh or connect clusters
-  configured using the Cilium CLI operating in *classic* mode, as the two modes are
-  not compatible with each other.
```

Comment:
I'd personally preserve this warning about using the new CLI version against older installations, as it can be a source of confusion and lead to multiple errors.